lib/proxy-connection.js

```
@@ -29,19 +29,23 @@ ProxyConnection.prototype._installListeners = function() {
   this.browserConnection.on('message', function(request) {
     self.processMessage(request).then(function() {
       self.janusConnection.send(request);
-    })
+    }).catch(function(error) {
+      //error has been caught and logged by processMessage
+    });
```

I'm not sure about node conventions and best practices, but isn't ignoring exceptions a bad practice?
It errors are caught in `processMessage` could empty `catch` block be deleted?